text/0000-default-scoped-light.md

```diff
@@ -0,0 +1,133 @@
+
+<details><summary>Implementation details</summary>
+
+Currently we use `*.scoped.css` as a signal for whether a stylesheet should be scoped or not, and the `@lwc/template-compiler` appends a [`?scoped=true` suffix](https://github.com/salesforce/lwc/blob/cfee90b9b2f03fe0242cd1366b394e0ee7373227/packages/%40lwc/compiler/src/transformers/template.ts#L86-L87) to the URL of the import to signal this. E.g.:
+
+```js
+import _implicitStylesheets from './component.css';
+import _implicitScopedStylesheets from './component.scoped.css?scoped=true';
+```
+
+In the new system, the `?scoped=true` suffix would vary depending on whether the importer is a light DOM or shadow DOM template. In light DOM, it would look like this:
+
+```js
+import _implicitStylesheets from './component.css?scoped=true';
+import _implicitScopedStylesheets from './component.scoped.css?scoped=true';
+import _implicitUnscopedStylesheets from './component.scoped.css';
+```
+
+...whereas in shadow DOM, it would be:
+
+```js
+import _implicitStylesheets from './component.css';
+import _implicitScopedStylesheets from './component.scoped.css?scoped=true';
+import _implicitUnscopedStylesheets from './component.scoped.css';
+```
+
+</details>
+
+## Drawbacks
+
+### Breaking existing behavior
+
+Light DOM is considered "beta" and has not officially shipped in LEX yet. However, some folks may be using it off-core. In those cases, the behavior will change, and they will have to fix their existing components, if they were relying on `*.css` being unscoped for light DOM components.
+
+### More complexity in the design
+
+The old system (`*.css` is unscoped, `*.scoped.css` is scoped) was simple to describe. The new system has a 2x3 table to explain how it works.
+
+
+Arguably the new system is "simpler," in that developers can mostly just use `*.css` and it will be "scoped." However, once you get into the weeds of what "scoped" actually means (e.g. the fact that scoping is useful for shadow DOM as well, e.g. in the case of a shadow-parent-light-child situation or [multiple templates](https://lwc.dev/guide/html_templates#render-multiple-templates)), then the existing system is actually much simpler.
+
+### Breaking changes for bundlers/adapters
+
+We will need to change the existing behavior in the LWC adapters for Rollup, Jest, Webpack, LWR, lwc-platform, and anywhere else that implements the LWC resolution algorithm. We may break existing unit tests and apps while getting the feature out the door. 
+
+## Alternatives
+
+### Make scoped styles the default everywhere
+
+We cannot do this, because it would potentially break existing shadow DOM components. Even forgetting that light DOM exists, the addition of an extra class selector could change the specificity of the CSS rules, and thus the runtime styling behavior (in both synthetic and native).

```

Comment:
@gonzalocordero Does this section address [your concern](https://github.com/salesforce/lwc-rfcs/pull/65#issuecomment-1176592916)?